plugins/search/api-report.md

```diff
@@ -75,6 +75,11 @@ export type HomePageSearchBarProps = Partial<
   Omit<SearchBarBaseProps, 'onChange' | 'onSubmit'>
 >;
 
+// Warning: (ae-missing-release-tag) "rootRouteRef" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const rootRouteRef: RouteRef<undefined>;
```

Comment:
If this is the first time we're exporting this from the plugin, perhaps we could give it a more semantic name than `rootRouteRef`?  Maybe `searchPageRef` or something?  It's tiny, but it makes route resolution code in consuming plugins and apps so much easier to read/understand:

```ts
const getSearchPath = useRouteRef(searchPageRef);

<Link to={getSearchPath()}>Search All</Link>
```

Reply:
Don't export it! It's already available as `searchPlugin.routes.root`

Reply:
Nice, thanks @Rugvip will update!